### ISSUE TITLE
Running ./bin/md2resume results in error message - Linux

### ISSUE THREAD

**Issue by ejwaibel** [there4/markdown-resume#70]
```
$ ./bin/md2resume 

The dependency '' was not found. Please run `composer install` to install dependencies.
```

I have run `composer install` as instructed and all dependencies have been installed, but still receive the error message. This is happening in Ubuntu.

**Comment by orxvan** [comment]
yeah,I receive it also

**Comment by abelsonlive** [comment]
+1

**Comment by kaaljabr** [comment]
I found the issue it is line number 13 of md2resume file

```
$autoloadPath = basename($baseDir) === 'markdown-resume'
              ? realpath($baseDir . '/../../autoload.php')
              : $baseDir . '/vendor/autoload.php';
```
executing this piece of code sets null or undefined in $autoloadPath.

change it to 
```
$autoloadPath = $baseDir . '/vendor/autoload.php';
```
it should work

but make sure autoload.php file exists in ./vendor

**Comment by spawnia** [comment]
Should by resolved by #79
